client/src/api/v3/boost.ts

```diff
@@ -0,0 +1,30 @@
+import { emit } from 'api/common/socket';
+import { BoostConfigType } from 'types/boost';
+
+/**
+ * Send the content of the given configuration file on `boost/configs/action` over MQTT
+ *
+ * @param type the type of the configuration being sent
+ * @param configFiles list of files, only the first would be considered
+ */
+export default function uploadConfig(
+  type: BoostConfigType,
+  configFiles: FileList,
+) {
+  const topic = 'send-uploaded-config';
+  const reader = new FileReader();
+
+  // Called when FileReader has completed reading a file
+  reader.onload = () => {
+    console.log(reader.result);
+    const payload = {
+      action: 'upload',
+      configType: type,
+      configName: configFiles[0].name,
+      content: reader.result,
+    };
+    emit(topic, JSON.stringify(payload).replace(/\\n/g, ''));
```

Comment:
Why do you remove the newlines? I reckon they'd be good to keep, so the stored configs are more readable

Reply:
Good point! I only removed them cos they looked ugly on mqtt (since this was the only placed I looked), completely forgot at the time that `boost` would store it directly.